EventListener/ValidationRequestListener.php

```diff
@@ -88,8 +91,9 @@ public function onKernelRequest(GetResponseEvent $event)
             $dataObject = new \stdClass();
         }
 
-        // Validate data with given schema.
-        $validator = new \JsonSchema\Validator();
+        $schemaStorage = new SchemaStorage();
+        $schemaStorage->addSchema('file://' . $schemaFile, $schema);
```

Comment:
if you have a lot of references in you schema it would be useful to cache the resulting-schema (resolved references) - would that be possible with this storage?